src/Math/Direction2.cs

```diff
@@ -245,6 +245,12 @@ public override int GetHashCode()
 
         #endregion
 
+        #region tostring
+
+        public override string ToString() => $"{Radians} rad";
```

Comment:
Ditto. Also an option: Haven an overload with a parameter (create an enum) to either print degrees or radians or radians with the factor of pi extracted.